deploy/crds/integreatly_v1alpha1_redis_cr.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: integreatly.org/v1alpha1
+kind: Redis
+metadata:
+  # name must be between 1-40 characters
+  name: example-redis
+spec:
+  # i want my redis storage information output in a Secret named test in the Namespace cloud-resource-operator
+  secretRef:
+    name: redis-test
```

Comment:
do we want to allow this to target another namespace for the secret? For example ```openshift-3scale ```

Reply:
We had that initially but realised it allowed users to create a secret in any random namespace. We're just creating it in the product namespace now

Reply:
@maleck13 As the RHMI operator would be creating the CR and reading the secret to then populate whatever the product needs, I don't see the reason for allowing multiple namespaces involved?

Reply:
The main reason is to allow the product to use the secret directly rather than having to copy that secret around.  It would work similar to the credential minter. The controlling factor here is having permission to create the resource in the operators namespace.